components/FeedList/FeedList.jsx

```diff
@@ -30,24 +40,33 @@ const FeedList = ({state, navigation, loading, showHide}) => {
       );
     }
   };
+
+  const dataOfList = useMemo(
+    () => (route.name === 'Feed' ? filteredData : feedData),
+    [route.name, filteredData, feedData],
+  );
```

Comment:
not expansive function, no need for useMemo

Reply:
Fixed , but lets discuss this part tomorrow , please.